File: 新建文件夹/pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/brandService.js

    app.service("brandService",function ($http) {
    this.findAll=function () {
        return $http.get("/brand/findall.do");
    }
    this.findPage=function (page,rows) {
        return $http.get('/brand/findpage.do?page=' + page + '&rows=' + rows);
    }
    this.update=function (save) {
        return $http.post("/brand/update.do",save);
    }
    this.add=function (add) {
        return $http.post("/brand/add.do",add );
    }
    this.finOne=function (id) {
        return $http.get("/brand/findOne.do?id="+id+"");
    }
    this.dele=function (ids) {
        return $http.get("/brand/delete.do?ids="+ids);
    }
    this.search=function (page,rows,searchEntity) {
            return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
    }
        this.selectOptionList=function () {
            return $http.get("/brand/selectOptionList.do");
        }
});

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.entity.PageResult;
import com.pinyougou.pojo.TbBrand;


import java.util.List;
import java.util.Map;

/*
品牌信息
* */
public interface BrandService {
    // 查询所有品牌信息
    List<TbBrand> findAll();

    //返回分页列表
    PageResult findPage(Integer pageNum, Integer pagesize);

    // 增加品牌
    void add(TbBrand tbBrand);

    // 根据id查询
    TbBrand findOne(Long id);

    // 修改
    void  update(TbBrand tbBrand);

    // 删除品牌
    void delete(Long [] ids);

    // 按条件查询
    PageResult findPage(TbBrand tbBrand,int pageNum, int pagesize);

    List<Map> selectOptionList();

}
